.github/workflows/maven-ci.yml

```diff
@@ -49,14 +49,10 @@ jobs:
           ${{ runner.os }}-maven-
     # Some tests need screen access
     - name: Install xvfb
-      run: sudo apt-get install xvfb
+      run: sudo apt-get install -y xvfb
     # SonarQube scan does not work for forked repositories
     # See https://jira.sonarsource.com/browse/MMF-1371
```

Comment:
I would extend the comment here to say that we run the SonarQube analysis only for the `master` branch.

Reply:
And point out that there is a separate workflow for pull requests.

Reply:
Good suggestion!